distribution/assembly.xml

```diff
@@ -62,7 +62,6 @@
           <outputDirectory>bin</outputDirectory>
           <includes>
             <include>ConfigMerge</include>
-            <include>Messages</include>
```

Comment:
Any other tool in the plan? To wrap the most common calls.

Reply:
As far as I'm concerned I am not a big fan of this because with large API it might be impossible to maintain (huge switch, lot of options, etc.). And most of the calls are 1 liners and can be performed by common tools e.g. `curl, wget, ...`.

Reply:
ok

Reply:
as long as the API is properly documented (pls file a new issue to track this) it should be fine to live without the CLI tool.